src/www/system_advanced_notifications.php

```diff
@@ -30,46 +30,12 @@
 require_once("system.inc");
 
 if ($_SERVER['REQUEST_METHOD'] === 'GET') {
-    // SMTP
-    $pconfig['disable_smtp'] = isset($config['notifications']['smtp']['disable']);
-    $pconfig['smtpipaddress'] = !empty($config['notifications']['smtp']['ipaddress']) ? $config['notifications']['smtp']['ipaddress'] : null;
-    $pconfig['smtpport'] = !empty($config['notifications']['smtp']['port']) ? $config['notifications']['smtp']['port'] : null;
-    $pconfig['smtpssl'] = isset($config['notifications']['smtp']['ssl']);
-    $pconfig['smtptls'] = isset($config['notifications']['smtp']['tls']);
-    $pconfig['smtpnotifyemailaddress'] = !empty($config['notifications']['smtp']['notifyemailaddress']) ? $config['notifications']['smtp']['notifyemailaddress'] : null;
-    $pconfig['smtpusername'] =!empty($config['notifications']['smtp']['username']) ? $config['notifications']['smtp']['username'] : null;
-    $pconfig['smtppassword'] = !empty($config['notifications']['smtp']['password']) ? $config['notifications']['smtp']['password'] : null;
-    $pconfig['smtpfromaddress'] = !empty($config['notifications']['smtp']['fromaddress']) ? $config['notifications']['smtp']['fromaddress'] : null;
     // System Sounds
     $pconfig['disablebeep'] = isset($config['system']['disablebeep']);
```

Comment:
sorry to be picky... remaining options must go to system: settings: misc

Reply:
Oh, no problem at all - I was thinking something similar.

Reply:
❤️